cmd/fleetctl/gitops_test.go

```diff
@@ -238,6 +241,7 @@ org_settings:
     org_logo_url_light_background: ""
     org_name: ${ORG_NAME}
   secrets:
+software:
```

Comment:
Good question.  The PR assumes that `software` is an optional key.  However because of the declarative nature of gitops, declaring empty software will delete any existing installable software out of "no teams".  Is that ok @marko-lisica ?

Reply:
To note, this mimics the current behavior of teams software

Reply:
discussed answer, for reference: https://github.com/fleetdm/fleet/issues/19550#issuecomment-2261165089